Tools/Integration.cpp

```diff
@@ -17,20 +17,60 @@
 namespace ComPWA {
 namespace Tools {
 
-double integrate(ComPWA::Intensity &intensity,
+struct KahanSummation {
+  double sum;
+  double correction;
+  operator double() const { return sum; }
```

Comment:
I would not use the cast, but instead return this KahanSummation from the accumulate.

Reply:
Is there a disadvantage?

Reply:
I think its harder to understand than without. At least I asked myself what that is for and had to look through the code

Reply:
After summation you use it as a `double` so a cast seems fine to me. To understand what the code does it is not necessary to understand the details of KahanSummation. I prefer it this way.